section3/reflection.md

```diff
@@ -1,17 +1,26 @@
 ## Section 3 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+  1. I hadn't heard of SMART goals before. Setting specific, measurable, achievable, relevant, & time-bound goals would make my own goals more realistic & achievable.
+  2. Having a growth mindset makes it seem like one would be unafraid of any problem they encounter. Learning from one's mistakes & actively seeking out challenges would definitely by tougher to do than sitting back & waiting for something to happen.
 
 1. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+   I'm not afraid to ask questions & I try to learn from my mistakes as much as I can. However, I don't actively seek out challenges & that's where I need to improve my growth mindset.
```

Comment:
You will have plenty of time to hone that "seek out challenges" mindset during Turing
Suggestion for seeking solo work:
- During Turing => Focus on solo work being pre-learning, studying, prepping for assessments, managing your habits/time, looking ahead in the Mod
- After Turing => Work on whatever your heart desires. I've worked on personal websites, new languages/frameworks, building fun things (I've built out games and gui apps), etc